core/src/stack/outputs.rs

```diff
@@ -84,6 +88,27 @@ impl StackOutputs {
     // PUBLIC ACCESSORS
     // --------------------------------------------------------------------------------------------
 
+    /// Returns the value located at the specified position on the stack
+    pub fn get_stack_item(&self, idx: usize) -> Option<Felt> {
+        self.stack.get(idx).map(|&felt| felt.into())
+    }
+
+    /// Returns the word located at the specified position on the stack
```

Comment:
Might be worth explicitly stating that it returns the words starting at the specified _Felt_ position to avoid misinterpretation.

Reply:
done